modules/es7.promise.finally.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+// https://github.com/tc39/proposal-promise-finally
+var $export = require('./_export');
+
+$export($export.P, 'Promise', {
+  'finally': function _finally(onFinally){
+    var handler = typeof onFinally === 'function' ? onFinally : function(){};
+    var C = Promise;
```

Comment:
Use `_species-constructor`. Also, what is `Promise`? We can't use this global variable, if you need it for something - you can use `core.Promise || global.Promise`.